screen/edit-transaction/src/main/java/com/ivy/transaction/EditTransactionViewModel.kt

```diff
@@ -222,7 +225,11 @@ class EditTransactionViewModel @Inject constructor(
 
     @Composable
     private fun getTitleSuggestions(): ImmutableSet<String> {
-        return titleSuggestions.value
+        return if (showTitleSuggestions.value) {
```

Comment:
Just put `features.showTitleSuggestions.isEnabled(context)` here

Reply:
We cant do that because 
```
 suspend fun isEnabled(appContext: Context): Boolean =
        enabledFlow(appContext).first() ?: defaultValue
```
`isEnabled ` is a suspend function

Reply:
@akashs056 We can do:
```
features.showTitleSuggestions.asEnabledState(context)
```